internal/rmx.go

```diff
@@ -0,0 +1,69 @@
+package internal
```

Comment:
If we have the TUI client, would we want to share these enums? Can internal files be shared with the packages in `cmd`?

Reply:
Yes, there wouldn't be any instance where circular dependencies should occur.

Reply:
> If we have the TUI client, would we want to share these enums?

We could put enums inside a config repo which both client and server can import and use. But it should be a global file format like json so both clients can read and parse.